yarn.lock

```diff
@@ -16,11 +23,44 @@
   dependencies:
     "@babel/highlight" "^7.12.13"
 
+"@babel/code-frame@^7.16.7":
+  version "7.16.7"
+  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.16.7.tgz#44416b6bd7624b998f5b1af5d470856c40138789"
+  integrity sha512-iAXqUn8IIeBTNd72xsFlgaXHkMBMt6y4HJp1tIaK465CWLT/fG1aqB7ykr95gHHmlBdGbFeWWfyB4NJJ0nmeIg==
+  dependencies:
+    "@babel/highlight" "^7.16.7"
+
 "@babel/compat-data@^7.13.11", "@babel/compat-data@^7.13.12", "@babel/compat-data@^7.13.15", "@babel/compat-data@^7.13.8":
   version "7.13.15"
   resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.13.15.tgz#7e8eea42d0b64fda2b375b22d06c605222e848f4"
   integrity sha512-ltnibHKR1VnrU4ymHyQ/CXtNXI6yZC0oJThyW78Hft8XndANwi+9H+UIklBDraIjFEJzw8wmcM427oDd9KS5wA==
 
+"@babel/compat-data@^7.16.4":
+  version "7.17.0"
+  resolved "https://registry.yarnpkg.com/@babel/compat-data/-/compat-data-7.17.0.tgz#86850b8597ea6962089770952075dcaabb8dba34"
+  integrity sha512-392byTlpGWXMv4FbyWw3sAZ/FrW/DrwqLGXpy0mbyNe9Taqv1mg9yON5/o0cnr8XYCkFTZbC1eV+c+LAROgrng==
+
+"@babel/core@^7.1.0", "@babel/core@^7.12.3", "@babel/core@^7.7.2", "@babel/core@^7.8.0":
```

Comment:
Babel added a lot of dependencies :(

If there's a way to avoid adding it, while still having TS tests, I'm all ears

Reply:
Sorry, why do we need Babel at all? The `umbra-js` package has TS tests but doesn't use babel

Reply:
> The umbra-js package has TS tests but doesn't use babel

Yeah, good point. Seems like umbra-js runs the tests through hardhat. I'm not sure how that works with TS.

When I initially tried to run the TS tests, they all failed with Syntax errors b/c none of the typescript of ES6 idioms were being interpretted. I eventually found [this](https://jestjs.io/docs/getting-started#using-typescript) in the Jest docs about using TS with Jest, and it said that you should install `@babel/preset-typescript`, so I just blindly did that. But looking past the first paragraph of the docs, I see that there's another option, `ts-jest`, which seems much more lightweight. I'll try that.

Reply:
Seems to work well! Any idea what this warning is? I don't see any async calls in the tests
<img width="1289" alt="image" src="https://user-images.githubusercontent.com/17163988/157354916-45f2c329-ecea-41c1-9339-bed8e18e644e.png">

Reply:
Chatted about this in standup 👍